src/pl/senla/hotel/controller/ConsoleCollection.java

```diff
@@ -3,92 +3,210 @@
+        Guest guest3 = new Guest(3, "Adam", 13564899);
+        Guest guest4 = new Guest(4, "Helen", 905350235);
+        Guest guest5 = new Guest(5, "Buddy", 905350124);
+        Guest guest6 = new Guest(6, "Steve", 2834595);
+        Guest guest7 = new Guest(7, "Jim", 624339229);
+        Guest guest8 = new Guest(8, "Olga", 33343466);
+        Guest guest9 = new Guest(9, "Violet", 22222222);
+        Guest guest10 = new Guest(10, "Eugene", 3333333);
+        Guest guest11 = new Guest(11, "Sally", 44444444);
 
-        System.out.println("\n----- Client -----");
-
-        Client client0 = new Client(0, "Tom", 351434000);
-        Client client1 = new Client(1, "John", 380452004);
-        Client client2 = new Client(2, "Sam", 905350895);
-
-        System.out.println(CONSOLE_CREATE_CLIENT + clientController.create(client0));
-        System.out.println(CONSOLE_CREATE_CLIENT + clientController.create(client1));
-        System.out.println(CONSOLE_CREATE_CLIENT + clientController.create(client2));
-        System.out.println(clientController.read(0));
-        System.out.println(clientController.read(1));
-        System.out.println(clientController.read(2));
-        System.out.println(CONSOLE_READ_ALL_CLIENTS + clientController.readAll());
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest0));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest1));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest2));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest3));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest4));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest5));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest6));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest7));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest8));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest9));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest10));
+        System.out.println(CONSOLE_CREATE_GUEST + guestController.create(guest11));
+        System.out.println(guestController.read(0));
+        System.out.println(guestController.read(1));
+        System.out.println(guestController.read(2));
+        System.out.println(CONSOLE_READ_ALL_GUESTS + guestController.readAll());
 
-        client0.setPhoneNumber(378340012); //update Client
-        System.out.println(CONSOLE_CHANGE_CLIENT + clientController.update(client0));
-        System.out.println(clientController.read(0));
-        System.out.println(CONSOLE_READ_ALL_CLIENTS + clientController.readAll());
+        System.out.println("\n============================");
+        System.out.println("UPDATE GUEST");
+        guest0.setPhoneNumber(378340012); //update Client
```

Comment:
It will work here, because link;s will be same and you working in one thread. But for future remember - you must receive entity you want to change from entity holder. 
No need to fix it, just reminder for future